.github/workflows/metadata curator.yaml

```diff
@@ -0,0 +1,147 @@
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: backend
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dependencies
+        run: npm ci
+      - name: audit
+        if: github.event_name != 'release' || github.event.action != 'created'
+        run: npm audit
+  
+  unit_test:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: backend
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dependencies
+        run: npm ci
+      - name: test
+        run: |
+          cp config/default.json.example config/default.json
+          npm test
+
+  quay_io_backend:
+    runs-on: ubuntu-latest
+    needs: ["unit_test", "audit"]
+    defaults:
+      run:
+        shell: bash
+        working-directory: backend
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: push to quay
+        run: |
+          export BRANCH=${GITHUB_REF##*/}
+          chmod ugo+x ./docker_quayio_push
+          ./docker_quayio_push metadata_curator_api
+  
+  quay_io_frontend:
+    runs-on: ubuntu-latest
+    needs: ["quay_io_backend"]
+    defaults:
+      run:
+        shell: bash
+        working-directory: frontend
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: push to quay
+        run: |
+          export BRANCH=${GITHUB_REF##*/}
+          chmod ugo+x ./docker_quayio_push
+          ./docker_quayio_push metadata_curator_ui
+
+  
+  deploy_backend:
+    runs-on: ubuntu-latest
+    needs: ["quay_io_backend"]
+    if: success() && ( (github.event_name == 'release' && github.event.action == 'created') || (github.ref == 'refs/heads/master') || (github.ref == 'refs/heads/develop') )
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_DEPLOY_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_DEPLOY_PASSWORD }}
+    defaults:
+      run:
+        shell: bash
+        working-directory: backend
+    steps:
+      - uses: actions/checkout@v2
+      - name: push to dockerhub
+        run: |
+          export BRANCH=${GITHUB_REF##*/}
+          chmod ugo+x ./docker_push
+          ./docker_push metadata_curator_api
+
+  deploy_frontend:
+    runs-on: ubuntu-latest
+    needs: ["deploy_backend"]
+    if: success() && ( (github.event_name == 'release' && github.event.action == 'created') || (github.ref == 'refs/heads/master') || (github.ref == 'refs/heads/develop') )
+    env:
+      DOCKER_USERNAME: ${{ secrets.DOCKER_DEPLOY_USERNAME }}
+      DOCKER_PASSWORD: ${{ secrets.DOCKER_DEPLOY_PASSWORD }}
+      WEBHOOK_URL: ${{secrets.WEBHOOK_URL}}
+    defaults:
+      run:
+        shell: bash
+        working-directory: frontend
+    steps:
+      - uses: actions/checkout@v2
+      - name: push to dockerhub
+        run: |
+          export BRANCH=${GITHUB_REF##*/}
+          chmod ugo+x ./docker_push
+          ./docker_push metadata_curator_ui
+
+  # integration_tests:
```

Comment:
Any reason this step is commented out? Might be helpful to throw a comment there.

Reply:
It is commented out as the Katalon tests I ported over from travis but they don't run here. They are being replaced with nightwatch tests which will likely come with the other updates in regards to your other comments here!